src/server/services/wms/qgsmaprendererjobproxy.cpp

```diff
@@ -48,8 +55,54 @@ namespace QgsWms
     }
   }
 
+  void QgsMapRendererJobProxy::fetchSvgSymbol( QgsSymbol *symbol )
+  {
+    for ( QgsSymbolLayer *sl : symbol->symbolLayers() )
```

Comment:
Range based for loops must not be done on an rvalue (or it will detach)

```suggestion
    const auto symbolLayers = symbol->symbolLayers();
    for ( QgsSymbolLayer *sl : symbolLayers )
```